game/templates/game/match_history.html

```
@@ -0,0 +1,892 @@
+            <a href="/history/" style="color:var(--text)">History</a>
+        </nav>
+    </header>
+
+    <!-- ─── Main layout ─────────────────────────────────────────────────── -->
+    <div class="layout">
+
+        <!-- ── Sidebar: game list ──────────────────────────────────────── -->
+        <aside class="sidebar">
+            <div class="sidebar-header" style="display:flex; justify-content:space-between; align-items:center;">
+                <span>Recent games &nbsp;·&nbsp; last 48 hours</span>
+                <button class="btn-icon" style="width:auto; padding:4px 8px; font-size:0.75rem; border:none;" onclick="importPgn()" title="Import PGN">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)